test/shaderdb/ObjSharedVariable_TestArrayCopy_lit.comp

```diff
@@ -30,10 +30,10 @@ void main()
 ; RUN: amdllpc -spvgen-dir=%spvgendir% -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
 
 ; SHADERTEST-LABEL: {{^// LLPC}} SPIRV-to-LLVM translation results
-; SHADERTEST: @{{[0-9]}} = addrspace(3) global { i32, [16 x i32] }
-; SHADERTEST: store i32 %{{[0-9]*}}, i32 addrspace(3)* getelementptr inbounds ({ i32, [16 x i32] }, { i32, [16 x i32] } addrspace(3)* @{{[0-9]}}, i32 0, i32 1, i32 {{[0-9]*}})
-; SHADERTEST: %{{[0-9]*}} = load i32, i32 addrspace(3)* getelementptr inbounds ({ i32, [16 x i32] }, { i32, [16 x i32] } addrspace(3)* @{{[0-9]}}, i32 0, i32 1, i32 {{[0-9]*}})
-; SHADERTEST: %{{[0-9]*}} = getelementptr { i32, [16 x i32] }, { i32, [16 x i32] } addrspace(3)* @{{[0-9]}}, i32 0, i32 1, i32 %{{[0-9]*}}
+; SHADERTEST: @[[LDS:[^ ]*]] = addrspace(3) global { i32, [16 x i32] }
```

Comment:
Change all LDS to lds

Reply:
Hi Rex. It is conventional to use all-upper-case names for llvm-filecheck captured variables.

Reply:
OK. Thanks for pointing out this. Xue will merge the test change and code change together. Otherwise, GitHub testing will fail.